testing/triggers/jobs.merge.yaml

```diff
@@ -0,0 +1,9 @@
+name: jobs-merge
+filename: testing/jobs.merge.cloudbuild.yaml
+github:
+  name: cloud-run-hello
+  owner: GoogleCloudPlatform
+  push:
+    branch: ^master$
```

Comment:
s/master/main/ ?

Reply:
Will update once we migrate this repo to main. To avoid any accidents I'll migrate later